Remotion/Web/Core/Res/Themes/NovaViso/HTML/FormGrid.css

```
@@ -1,105 +1,95 @@
 /*  Style applied to the form grid table's outer structure  */
-table.formGridTable
+.formGridTable
 {
   border-collapse: collapse;
-  border-width: 0;
   width: 100%;
+  background-color: var(--color-standard-background);
 }
 
-td.formGridTitleCell,
-td.formGridSubTitleCell
+.formGridTitleCell,
+.formGridSubTitleCell
 {
   font-weight: bold;
   font-size: 125%;
-  padding: 2px 2px 2px 2px;
-  border: none;
-}
-
-td.formGridTitleCell h1
-{
-  width: 100%;
-  border-bottom: solid 1px #AAAAAA;
-  color: Black;
+  padding: calc(var(--remotion-themed-spacing) * 2) 0;
 }
 
 /*  Style applied to the cells containing the labels  */
-td.formGridLabelsCell
+.formGridLabelsCell
 {
-  background-color: #F0F0F0;
-  text-align: right;
-  padding: 0.3em 1em 0.3em 1em;
-  border-top: none;
-  border-left: none;
-  border-right: none;
-  border-bottom: solid 1px #C8C8C8;
-  width: 17%;
+  box-sizing: border-box;
+  width: 5%;
+  padding-top: var(--remotion-themed-spacing);
+  padding-right: var(--remotion-themed-spacing);
+  padding-bottom: var(--remotion-themed-spacing);
+  padding-left: 0;
```

You could write it that way, since all properties except `padding-left` have the same value.

```suggestion
  padding: var(--remotion-themed-spacing);
  padding-left: 0;
```